schema_based_on_diagram.sql

```diff
@@ -0,0 +1,44 @@
+
+CREATE TABLE patients (
+	id SERIAL PRIMARY KEY,
+	name varchar(22),
+	date_of_birth DATE
+);
+
+CREATE TABLE medical_histories (
+	id SERIAL PRIMARY KEY,
+	admitted_at timestamp,
+	patient_id int,
+	status varchar(22)
+);
+
```

Comment:
* Kindly add the remaining `foreign keys`  as based on the diagram you should have 6️⃣  `foreign keys` 
* HINT 💭 if you check each diagram you will find a `column id` with a table name ie ( `patient_id` which is referencing the` patients table`) create foreign keys for all such foreign keys 👍